vgen/src/main/java/io/gapi/vgen/csharp/CSharpGapicCodeLanguageProvider.java

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gapi.vgen.csharp;
+
+import com.google.api.tools.framework.model.Interface;
+import com.google.api.tools.framework.model.Model;
+
+import io.gapi.vgen.ApiConfig;
+
+public class CSharpGapicCodeLanguageProvider extends CSharpGapicLanguageProvider<Interface> {
+
+  private final CSharpInterfaceViewing viewing = new CSharpInterfaceViewing();
```

Comment:
I agree that having <Language>x<ProtoElement>Viewing isn't ideal, since there are a large number of possible combinations. (Note, however, that the languages other than C# and Python don't have any special per-protoElement-subclass behavior, so there is no need for any JavaXXXViewing.) However, I'm not sure specifying two classes in the config file helps this situation -- wouldn't we still need to have this file?